### ISSUE TITLE
make class diagram (Story 1)

### ISSUE THREAD

**Issue by MartijnAB** [MartijnAB/from-toxicological-data-to-toxicological-information#16]

**Comment by mkutmon** [comment]
that link doesn't work for me. can you add the png file here as a comment please?

**Comment by mkutmon** [comment]
![image](https://user-images.githubusercontent.com/2158343/52284319-78b35d80-2964-11e9-90b6-0c2a8b3658a3.png)

**Comment by mkutmon** [comment]
You can also copy it in here. 
Couple of small comments:
- execute_sparql_query is a general execution function for all WikiPathways SPARQL query, right?
- Is it needed to restrict the "make_chemical_compound" function to CAS numbers? What if a metabolite in a pathway doesn't have a CAS number? Is it then not possible to create a ChemicalCompound? 
- Just to double check - the aggregation goes from ChemicalCompound to Pathway, right? So a Pathway as ChemicalCompounds. 
![image](https://user-images.githubusercontent.com/2158343/52284729-45250300-2965-11e9-8b8b-ad0cc3aa50dc.png)

**Comment by MartijnAB** [comment]
wat te doen als het enige indicatie nummer van een chemiestof uit meerder mogelijkheden bestaat?
ik kan ze natuurlijk altijd in een lijstje stoppen. maar dat zal dat voor elk soort indicatie nummer moeten, wat mij een verkeert onderbuik gevoel geeft.

**Comment by mkutmon** [comment]
Yes, this is a very tricky issue with chemicals. We can discuss it in the sprint meeting since Denise has a lot of experience with this.

**Comment by mkutmon** [comment]
decision: let's start with only getting the Wikidata id from the Metabolites from the WP SPARQL endpoint.
